### ISSUE TITLE
not able to connect to sql server after creating container

### ISSUE THREAD

**Issue by nischaygowda** [Microsoft/mssql-docker#252]
I used the exact same container and exported DB_HOST, DB_USERNAME and DB_PASSWORD but still I am not able to connect to sql server. Its says login timeout expired. am I missing anything here?

However if the run the connect.py outside the docker container, it works fine. 

is there is any network issue that it is not able to reach the SQL server? How do I solve this?

**Comment by twright-msft** [comment]
@nischaygowda - Are you trying to do this with SQL Server or with MySQL?  The SQL Server containers don't use DB_HOST, DB_USERNAME, DB_PASSWORD like MySQL probably does.

**Comment by fenchu** [comment]
He is using: the pyodbc version: https://github.com/Microsoft/mssql-docker/blob/master/oss-drivers/pyodbc/README.md

**Comment by nischaygowda** [comment]
yes that's right.  the docker container works if I use the "network = host" option.
